src/components/victory-scatter/helper-methods.js

```diff
@@ -25,23 +25,25 @@ export default {
       };
 
       const text = this.getLabelText(props, datum, index);
-      const labelStyle = this.getLabelStyle(style.labels, dataProps) || {};
-      const labelProps = {
-        style: labelStyle,
-        x,
-        y: y - (labelStyle.padding || 0),
-        text,
-        index,
-        scale,
-        datum: dataProps.datum,
-        textAnchor: labelStyle.textAnchor,
-        verticalAnchor: labelStyle.verticalAnchor || "end",
-        angle: labelStyle.angle
-      };
-      childProps[eventKey] = {
-        data: dataProps,
-        labels: labelProps
-      };
+      if (!text && !props.events) {
```

Comment:
I think this is actually backwards. The stuff in this if block should _not_ happen `!text && !props.events`